themes/origins/styles/base/_common.scss

```
@@ -64,17 +65,30 @@ button,
     color: var(--color-gray-500);
     background-color: var(--color-gray-50);
   }
+  &:hover:is(.secondary) {
+    background-color: var(--color-base-black);
+    color: var(--color-base-white);
+    border-color: var(--color-base-black);
+  }
   &:hover:is(.primary, .destructive, .loading, .with-icon) {
     opacity: 0.8;
   }
-  &:hover:is(.secondary, .outline, .ghost, .icon) {
+  &:hover:is(.outline, .ghost, .icon) {
     background-color: var(--color-gray-50);
   }
   &:hover:is(.link) {
     text-decoration: underline;
   }
 }
 
+// LINKS
+a,
+.link {
+  &:hover {
+    text-decoration: underline;
+  }
+}
+
```

might this change affect all theme links (check `header` & `footer` links)

```suggestion
// LINKS
.link:hover {
  text-decoration: underline;
}
```